### PR TITLE
configure workload api with new workloads api

### PR DESCRIPTION
**Pull request by inyee786**
Signed-off-by: inyee786 <intakhab.ali@mayadata.io>

**What this PR does / why we need it**:

this pull request fix the multiple call of cluster to one call

change the variable name for agile api

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/services/kubernetes.service.ts [hunk 1]
```diff
@@ -1,70 +1,83 @@
...
-            params:{
-                workloadname:workloadname,
-                openebsengine:openebsengine
-            }
-        });
-    }
+   private apiurl: string;
+   private host: string;
+   private clusterDomain:any;
+   // private workloadapi = `http://localhost:4000/`
```
Comment (chandankumar4) [inline src/app/services/kubernetes.service.ts]: Remove this comment content

File: src/app/services/kubernetes.service.ts [hunk 2]
```diff
@@ -1,70 +1,83 @@
...
+
 
-    getPodDetails(appnamespace: string, volumenamespace: string) {
-        return this.http.get<overAllStatus>(this.apiurl + "pods/sequence", {
-            params: {
-                appnamespace: appnamespace,
-                volumenamespace: volumenamespace
-            }
+       this.host = window.location.host;
+       console.log(this.host);
```
Comment (chandankumar4) [inline src/app/services/kubernetes.service.ts]: Remove this line, not required

File: src/app/services/kubernetes.service.ts [hunk 3]
```diff
@@ -1,70 +1,83 @@
...
+  
+   setApiUrl(apiUrl: string){
+       this.apiurl = apiUrl;
+       console.log(this.apiurl);
+   }
+   getApiUrl(){
+       return this.apiurl;
+   }
+   getJivaVolumeDetails(workloadname:string,openebsengine:string) {
+       console.log(this.apiurl);
```
Comment (chandankumar4) [inline src/app/services/kubernetes.service.ts]: This line not required

File: src/app/services/kubernetes.service.ts [hunk 4]
```diff
@@ -1,70 +1,83 @@
...
+       return this.http.get(this.apiurl + "openebs/volume",{
+           params:{
+               workloadname:workloadname,
+               openebsengine:openebsengine
+           }
+       });
+   }
+
+   getPodDetails(appnamespace: string, volumenamespace: string) {
```
Comment (chandankumar4) [inline src/app/services/kubernetes.service.ts]: Remove the unused code

File: src/app/services/kubernetes.service.ts [hunk 5]
```diff
@@ -1,70 +1,83 @@
...
+       return this.http.get<overAllStatus>(this.apiurl + "pods/sequence", {
+           params: {
+               appnamespace: appnamespace,
+               volumenamespace: volumenamespace
+           }
+       });
+   }
+   getListNamespaces() {
```
Comment (chandankumar4) [inline src/app/services/kubernetes.service.ts]: Remove the unused function

File: src/app/workload-dashboard/workload-dashboard.component.ts [hunk 6]
```diff
@@ -1,62 +1,60 @@
+
 import { Component, OnInit } from "@angular/core";
 import { KubernetsService } from "../services/kubernetes.service";
 import * as $ from "jquery";
 import { Subscription, Observable, timer } from "rxjs";
 import { Meta,Title } from "@angular/platform-browser";
 import {
-  allApplication
+ allApplication
```
Comment (chandankumar4) [inline src/app/workload-dashboard/workload-dashboard.component.ts]: Make this in single line

File: src/app/workload-dashboard/workload-dashboard.component.ts [hunk 7]
```diff
@@ -1,62 +1,60 @@
+
```
Comment (chandankumar4) [inline src/app/workload-dashboard/workload-dashboard.component.ts]: Remove this line

File: src/app/workload-dashboard/workload-dashboard.component.ts [hunk 8]
```diff
@@ -1,62 +1,60 @@
...
-        });
-      });
-    });
-  }
-
-  setApiUrl(apiurl: string){
-    localStorage.setItem('apiurlkey' , apiurl);
-    this.kubernetsServices.setApiUrl(localStorage.getItem('apiurlkey'));
-  }
+   timer(0, 500000).subscribe(x => {
```
Comment (chandankumar4) [inline src/app/workload-dashboard/workload-dashboard.component.ts]: 500000 -> 300000

File: src/app/workload-dashboard/workload-dashboard.component.ts [hunk 9]
```diff
@@ -1,62 +1,60 @@
...
+       $("#myTable tr").filter(function() {
+         $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
+       });
+     });
+   });
+ }
+
+ setApiUrl(apiurl: string){
+   localStorage.setItem('apiurlkey' , apiurl);
+   console.log("hgvjhgvjhgvjhgjhgh")
```
Comment (chandankumar4) [inline src/app/workload-dashboard/workload-dashboard.component.ts]: Remove this line

File: src/app/services/kubernetes.service.ts (hunk 4 above)
Comment (inyee786) [reply src/app/services/kubernetes.service.ts]: this is required

**Pull request merged**